### PR TITLE
Add model for skins field of Order Model, add new root index.js entry…

### PR DESCRIPTION
**Pull request by danil-naidonov**
… file, add socker_url for WS config, update DOC

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "skinsback-sdk",
```
Comment (malsatin) [inline package.json:2]: why such changes?

**Comment by malsatin** [comment]
Could we add some default config parameters? For api url and socket url. It also implies moving skinsback api url as an optional config parameter

**Comment by danil-naidonov** [comment]
> Could we add some default config parameters? For api url and socket url. It also implies moving skinsback api url as an optional config parameter

Added default URLs for socket and api

File: package.json (hunk 1 above)
Comment (danil-naidonov) [reply package.json:2]: [solved the problem](https://github.com/shaitan-masters/skinsback-sdk/pull/3/commits/913dabcb9c165085306e961a30e6689f776816a6)

File: README.md [hunk 2]
```diff
@@ -126,15 +126,16 @@ import {WebSockets} from 'skinsback-sdk';
 ## Constructor options
 
 - `options[shop_id]`: your shop_id key **required**
-- `options[secret_key]`: Your personal `secret key` for generate signature. [More about generate signature](https://skinsback.com/profile.php?act=api&item=market_websocket).
-
-#### Example
+- `options[secret_key]`: Your personal `secret key` for generate signature. [More about generate signature](https://skinsback.com/profile.php?act=api&item=market_websocket) **required**.
+- `options[socket_url]`: SkinsBack socket url **required**. [Show more](https://skinsback.com/profile.php?act=api&item=market_websocket)
```
Comment (malsatin) [inline README.md]: now it is not required

File: src/API.ts [hunk 3]
```diff
@@ -21,7 +22,7 @@ class API {
     constructor(apiConfig: ApiConfig) {
         this.config = apiConfig;
         this.axios = axios.create({
-            baseURL: this.config.apiUrl || 'https://skinsback.com/api.php',
+            baseURL: this.config.apiUrl || API_URL,
```
Comment (malsatin) [inline src/API.ts:25]: apiUrl parameter is not added to the documentation

File: package.json [hunk 4]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "skinsback-sdk",
-  "version": "0.1.0",
-  "description": "SDK for integrations with skinsback.com",
-  "main": "js/index.js",
+  "version": "0.2.0",
+  "description": "skinsback.com API",
```
Comment (malsatin) [inline package.json]: could we remain current package description?

**Pull request merged**